staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1352,10 +1352,10 @@ type CustomizeComponents struct {
 
 // +k8s:openapi-gen=true
 type CustomizeComponentsPatch struct {
-	ResourceName string    `json:"resourceName,omitempty"`
-	ResourceType string    `json:"resourceType,omitempty"`
-	Patch        string    `json:"patch,omitempty"`
-	Type         PatchType `json:"type,omitempty"`
+	ResourceName string    `json:"resourceName"`
```

Comment:
Isn't this a change that is backwards incompatible?

Reply:
hmm, i guess it would be. How do we account for that?

Reply:
I figure that such changes are allowed (I thought that they weren't but when reading the v1 mail from David I understood that in general only field name changes are api changes for that matter). So we should at least add this to the release note. Don't know what else there would be to do. @rmohr is there?

Reply:
I have added it to the release note! thanks for catching that :)